File: Assignments/Assignment_18/p3.c
#include<stdio.h>
void main(){

        int row,alp=65;
        printf("Enter the no of rows :\n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=i;j>0;j--){

                        printf("%c\t",alp);
                }printf("\n\n");
                alp++;
        }
}

File: JAVA/9.Nested_For_2/p3.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		int counti=1,counto=1;

		for(int i=0;i<row;i++){
		
			counti=counto;
			for(int j=0;j<row;j++){
			
				System.out.print(counti+"\t");
				counti++;
			}
			counto++;
			System.out.println();
		}
	}
}
